docs/adr/0011-put-integration-tests-in-seperate-folder.md

```diff
@@ -0,0 +1,44 @@
+# Put Integration Tests in seperate source directory
+
+In order to get a better structure in the `server` module we want to reorganize the tests (Seperate Integration and Unit tests)
+
+## Considered Alternatives
+
+* Use JUnit Categories and the same `test` directory to seperate between Integration and Unit Tests
+* Store Integration and Unit Tests in seperate directories and use JUnit Categories to categorize them
+* Store Integration and Unit Tests in seperate directories and use a required Suffix on Integration Test Classes
+* Migrate project to gradle
+
+## Decision Outcome
+
+* Chosen Alternative: Store Integration and Unit Tests in seperate directories and use JUnit Categories to categorize them
+* While still Clearly seperating Unit and Integration tests this method still allows a fine grained categorisation of tests.
+* Consequences: The resource directory of the `ìntegration` folder has to be manually set within the IDE
```

Comment:
I think this is not a consequence: I didn't have to do it, was done automatically

Reply:
Lets say it might cause issues. Because as I said already i had to set it manually even after restarting the IDE